Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.0)
+    oj (3.10.5)
     orm_adapter (0.5.0)
     parallel (1.19.1)
     parser (2.6.5.0)
       ast (~> 2.4.0)
     passenger (6.0.4)
       rack
       rake (>= 0.8.1)
-    polyamorous (2.3.0)
-      activerecord (>= 5.0)
+    polyamorous (2.3.2)
+      activerecord (>= 5.2.1)
     psych (3.1.0)
-    public_suffix (4.0.1)
+    public_suffix (4.0.3)
     puma (4.3.3)
       nio4r (~> 2.0)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-cors (0.4.1)
     rack-proxy (0.6.5)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.2.1)
-      actioncable (= 6.0.2.1)
-      actionmailbox (= 6.0.2.1)
-      actionmailer (= 6.0.2.1)
-      actionpack (= 6.0.2.1)
-      actiontext (= 6.0.2.1)
-      actionview (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    rails (6.0.2.2)
+      actioncable (= 6.0.2.2)
+      actionmailbox (= 6.0.2.2)
+      actionmailer (= 6.0.2.2)
+      actionpack (= 6.0.2.2)
+      actiontext (= 6.0.2.2)
+      actionview (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       bundler (>= 1.3.0)
-      railties (= 6.0.2.1)
+      railties (= 6.0.2.2)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    rails_best_practices (1.19.4)
+    rails_best_practices (1.19.5)
       activesupport
-      code_analyzer (>= 0.4.8)
+      code_analyzer (>= 0.5.1)
       erubis
       i18n
       json
-      require_all (~> 2.0)
+      require_all (~> 3.0)
       ruby-progressbar
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rainbow (3.0.0)
     rake (13.0.1)
-    ransack (2.3.0)
-      actionpack (>= 5.0)
-      activerecord (>= 5.0)
-      activesupport (>= 5.0)
+    ransack (2.3.2)
+      activerecord (>= 5.2.1)
+      activesupport (>= 5.2.1)
       i18n
-      polyamorous (= 2.3.0)
+      polyamorous (= 2.3.2)
     rb-fsevent (0.10.3)
-    rb-inotify (0.10.0)
+    rb-inotify (0.10.1)
```

Comment:
### [rb-inotify](https://github.com/guard/rb-inotify)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.10.0 → 0.10.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/guard/rb-inotify/compare/v0.10.0...v0.10.1).

These are the individual commits:
- (3fae270) [Update documentation regarding resource limits. Fixes #61.](https://github.com/guard/rb-inotify/commit/3fae270905c7bd259e69fe420dbdcd59bb47fa3d)
- (469f873) [allow stopping a notifier from within an event callback](https://github.com/guard/rb-inotify/commit/469f873db45d1d4325546fad02b01ee0bbe63334)
- (4c3fde2) [Fixes #91.  Make IO resources not autoclose.](https://github.com/guard/rb-inotify/commit/4c3fde22535025d217ae503edf30013a1c4e1273)
- (1588952) [Limit changing auto close behavior off to just JRuby.  This …](https://github.com/guard/rb-inotify/commit/158895298f20b336e1f0717de5a4a67ad7769056)
- (e539d95) [Remove rbx-3.](https://github.com/guard/rb-inotify/commit/e539d9583b3a54ba3064768afc012c4175d14196)
- (82a50f8) [Bump version.](https://github.com/guard/rb-inotify/commit/82a50f8d106dc0af7e41fa19933849a0c9ce03ae)
- (59c2ea0) [Fix formatting.](https://github.com/guard/rb-inotify/commit/59c2ea08bdfb3af61f720e3bd1163db5960f3ccc)


</details>